client/src/app/auth/forgot-password/page.tsx

```diff
@@ -0,0 +1,126 @@
+"use client";
+import { useState } from "react";
+import useFetch from "@/src/hooks/useFetch";
+import { AuthService } from "@/src/services/AuthService";
+import Link from "next/link";
+import Navbar from "@/src/components/Navbar/Navbar";
+import Footer from "@/src/components/Footer/Footer";
+import useThemeStore from "@/src/store/themeStore";
+import Image from "next/image";
+import ForgotPasswordSvg from "@/public/assets/Login.svg";
```

Comment:
Svg's should be present inside icons folder of component. Reason, to create it as a component and also maintain react project structure.

Reply:
hey @abhijeetnishal for Icon i followed login page svg which was already there and followed same directory structure the 
Login page does. so should i add that svg inside "forgot-password folder" ??

Reply:
Ok then leave it for now.